source/SIL.AppBuilder.Portal.Frontend/src/data/orbitjs-operations-support/serialize-to-api.ts

```diff
@@ -0,0 +1,187 @@
+import {
```

Comment:
this stuff and all related stuff will (eventually) get pulled directly into orbit.js,